### ISSUE TITLE
Development branch outputs much less Pokemons

### ISSUE THREAD

**Issue by Scildrei** [AHAAAAAAA/PokemonGo-Map#538]
I have a running instance of the master code from today morning and it finds 36 Pokemons in my area, whereas the development branch outputs only 9 of them. 

I restarted the dev script multiple times, but I cannot find all the Pokemons from the master branch, even though I have no errors and I let it loop for ~5 minutes everytime.

The inconsistency is present anywhere on the map. I run both scripts with the same arguments and step 6.

**Comment by rezz3x** [comment]
You are right, also pokestop's and gym's aren't shown on the Map with the args -dg and -dp.

**Comment by Scildrei** [comment]
I believe pokestops and gyms aren't yet implemented.

**Comment by rezz3x** [comment]
Ah okay the nvm

**Comment by wcastello** [comment]
@Scildrei can you try my fork and see if the time fixes I did make it shows the same Pokemons to you? 
The dates on develop are still messed up, maybe the queries are filtering more pokemons than it should too. Try https://github.com/wcastello/PokemonGo-Map

**Comment by Scildrei** [comment]
@wcastello will try in a few minutes and let you know!

**Comment by Scildrei** [comment]
@wcastello Nope, same problem.

**Comment by fredgido** [comment]
I can confirm this.
